packages/voila/src/plugins/outputs/tools.ts

```diff
@@ -63,7 +62,7 @@ export function handleExecutionResult({
 }: {
   payload: IExecutionResultMessage['payload'];
   rendermime: IRenderMimeRegistry;
-  widgetManager: VoilaWidgetManager;
+  widgetManager: any;
```

Comment:
Could this have used the `type` approach too?

Reply:
Ah indeed I missed that one